i18n.ts

```diff
@@ -22,7 +25,7 @@ export const SUPPORTED_LANGUAGES = {
   ar: 'عربى',
   hi: 'हिंदी',
   kn: 'ಕನ್ನಡ',
-  ta: 'தமிழ்',
+  ta: 'ತಮಿಳು',
```

Comment:
For ta,  the value is not in tamil language. it should be 
தமிழ்

Reply:
Changes into respective language